app/views/planning_applications/neighbour_letters/_letter_template.html.erb

```diff
@@ -33,7 +33,7 @@
         </p>
       <% end %>
     </div>
-    <%= form.text_area :neighbour_letter_text, rows: 25, class: "govuk-body neighbour-letter-template" %>
+    <%= form.text_area :neighbour_letter_text, value: "#{I18n.t("neighbour_letter_template.#{planning_application.application_type.name}")}", rows: 25, class: "govuk-body neighbour-letter-template" %>
```

Comment:
Also not convinced this is the best way 😬

Reply:
I don't think the string interpolation is needed, if we're interpolating the entire string.

```suggestion
    <%= form.text_area :neighbour_letter_text, value: I18n.t("neighbour_letter_template.#{planning_application.application_type.name}"), rows: 25, class: "govuk-body neighbour-letter-template" %>
```